jbalsas/liferay-portal
# LPS-116361 Remote web apps scaffolding and PoC

Please see commit messages for a description of the functionalities contained in the PR. I suggest starting the review by looking at each commit first, instead of going to the `Files changed` section directly.

The way to test this is:

1) Start the portal (maybe deleting the DB first if the `remotewebappentry` table is not created on the first try).
2) Edit the home page (click the pencil)
3) In `Fragments and Widgets` click on the `Widgets` tab
4) Find the `SAMPLE` category
5) Drag and drop entry `El País` to the page

That should create a portlet which renders an `iframe` pointing to `http://www.elpais.com`.
